README.md

```diff
@@ -1,11 +1,11 @@
 ## Fetch Data 
 #### Call function to fetch data
-`prediction_data <- fetch_data('1_fetch/out/model_RMSEs.csv')`
+`prediction_data <- download_data('1_fetch/out/model_RMSEs.csv')`
```

Comment:
If you'd like, you could update these to `prediction_data <- scmake('1_fetch/out/model_RMSEs.csv')` but no need

Reply:
Could you then say `model_results<- scmake('2_process/out/model_summary_results.csv')`? Or does that not work because it needs the downloaded data?

Reply:
@jread-usgs (i.e would it throw an error if it was run out of sequence [before the data was downloaded])

Reply:
Yes, you can say `model_results<- scmake('2_process/out/model_summary_results.csv')` - scipiper will figure out what needs to go into producing that .csv file, whether it has it or needs to build it, and will run the function you defined (or not run it if it's already up to date!)